Source/HTTP/WinHttp/winhttp_http_task.cpp

```diff
@@ -863,6 +863,20 @@ HRESULT winhttp_http_task::send(
         return HRESULT_FROM_WIN32(dwError);
     }
 
+    if (!m_call->sslValidation)
+    {
+        DWORD dwOption = SECURITY_FLAG_IGNORE_UNKNOWN_CA | SECURITY_FLAG_IGNORE_CERT_CN_INVALID;
+        if (!WinHttpSetOption(
+            m_hRequest, 
+            WINHTTP_OPTION_SECURITY_FLAGS, 
+            &dwOption, 
+            sizeof(dwOption)))
+        {
+            DWORD dwError = GetLastError();
+            HC_TRACE_WARNING(HTTPCLIENT, "winhttp_http_task [ID %llu] [TID %ul] WinHttpSetOption errorcode %d", HCHttpCallGetId(m_call), GetCurrentThreadId(), dwError);
```

Comment:
are you intentionally eating this error here?  Should it be returned?

Reply:
Yes, I didn't feel like an error in setting that option was serious enough to prevent the request from going through. Do you prefer if we return in case of error?

Reply:
Yeah eating the error is fine for this